src/grammar.json

```diff
@@ -7759,7 +7759,7 @@
                     },
                     {
                       "type": "PATTERN",
-                      "value": "\\*[^\\/]"
+                      "value": "\\*[^/]"
```

Comment:
`grammer.js` wasn't in sync with the generated parser? Is that what happened?

Reply:
I think so; alternatively it might be the `tree-sitter` CLI version I’m using. Maybe @maxbrunsfeld will know? All the tests still pass so I’m not sure what the issue would have been.

Reply:
I think sometimes the exact escaping here depends on the version of Node.js that you use to generate the parser. This happens to me sometimes too, because I happen to have a different `node` in my path. At some point, it might be worth tweaking the Tree-sitter code to ensure this is consistent, but I think it's fine for now.